RemoteHealthcare/VRMethod.cs

```diff
@@ -758,5 +781,35 @@ public static void SetSkyBoxTime(ref Connection connection, float time)
             string skyboxSetTimeResponse = "";
             connection.SendViaTunnel(tunnelSetTimeJson, response => skyboxSetTimeResponse = response);
         }
+
+        public static void SetCamera(ref Connection connection, string bikeId)
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)